src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -16,20 +18,6 @@
     <SymbolPackageOutputPath>$(PackageOutputPath)</SymbolPackageOutputPath>
   </PropertyGroup>
 
-  <Target Name="SetTargetBasedPackageVersion"
-          BeforeTargets="GenerateNuSpec"
-          DependsOnTargets="GetProductVersions">
-    <PropertyGroup>
-      <Version>$(ProductVersion)</Version>
-      <!--
-        PackageVersion is normally calculated using Version during static property evaluation, but
-        we need some info from GetProductVersions, so it's too late to rely on that. We need to set
-        both in target evaluation, here.
-      -->
-      <PackageVersion>$(ProductVersion)</PackageVersion>
-    </PropertyGroup>
-  </Target>
-
```

Comment:
This might have been necessary in the past but AFAIK, NuGet calculates PackageVersion correctly, even for pkgprojs.